nodl_python/nodl/_parsing/_parsing.py

```diff
@@ -34,16 +34,32 @@ def _parse_interface(interface: etree._Element) -> List[Node]:
 
 
 def _parse_element_tree(element_tree: etree._ElementTree) -> List[Node]:
-    """Extract an interface element from an ElementTree if present."""
+    """
+    Extract an interface element from an ElementTree if present.
```

Comment:
This comment applies throughout: according to [PEP-257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) docstrings shouldn't start with a newline.

Reply:
PEP-257 allows either, pydocstyle (what is used to check pep257) has two separate errors for either https://pep257.readthedocs.io/en/latest/error_codes.html

Error code D212 and D213. For some reason `colcon test` does not like _types.py:__iadd__ having a same line docstring. Troubleshooting...

Reply:
Per https://github.com/ament/ament_lint/blob/master/ament_flake8/ament_flake8/configuration/ament_flake8.ini D212 (same line docstrings) is explicitly ignored.

Likewise at https://github.com/ament/ament_lint/blob/master/ament_pep257/ament_pep257/main.py#L50.

Pydocstyle itself disables BOTH by default, so I'm not sure why either is running in the first place. https://pep257.readthedocs.io/en/latest/error_codes.html#default-checks

Since we're removing iadd anyway I won't look any further, but I don't know what was causing this.